builder/dockerfile/builder.go

```diff
@@ -324,6 +324,9 @@ func (b *Builder) build(stdout io.Writer, stderr io.Writer, out io.Writer) (stri
 	}
 
 	fmt.Fprintf(b.Stdout, "Successfully built %s\n", shortImgID)
+	for _, rt := range repoAndTags {
```

Comment:
I thought about it but wanted to minimize the modifications (as I'm a newbie in golang).
Moving the "Successfully built" before and displaying the created tags when they are created can be more relevant in case of error in one tagging: the image was successfully built and the first tags were created.

E.g. if there is an error while creating tag 'localhost:5000/test:latest', this:
```
[...]
 ---> Using cache
 ---> dc67a808676c
Successfully built dc67a808676c
Successfully tagged test:latest
Successfully tagged zigarn/test:latest
Successfully tagged test:1.0
Successfully tagged zigarn/test:1.0
ERROR
```
Would be better than this:
```
[...]
 ---> Using cache
 ---> dc67a808676c
ERROR
```